schemas/bridge/SCHEMA.md

```diff
@@ -0,0 +1,135 @@
+|-------------------------|-----------------------------------------------------------|--------|
+| timestamp                | The timestamp of the transaction.                         | number |
+| block_number             | The block number of the transaction.                      | number |
+| transaction_hash         | The hash of the transaction this bridge originated from.  | string |
+| source_chain_id          | The source standard chain ID.                             | number |
+| destination_chain_id     | The destination standard chain ID.                        | number |
+| volume_usd               | The volume of the transaction in USD.                     | number |
+| user_address             | The address of the user initiating the transaction.       | string |
+| solver_address           | The address of the solver (for intent-based bridges).     | string |
+| pool_address             | The address of the pool (for pool-based and mint-based bridges). | string |
+| bridge_type              | The type of bridge (pool-based, mint-based, or intent-based). | string |
+
+### Maker Snapshot
+
+List of makers in intent-based bridge protocols.
+
+| Property                | Description                                               | Type   |
+|-------------------------|-----------------------------------------------------------|--------|
+| timestamp                | The timestamp of the snapshot.                            | number |
+| user_address             | The address of the maker.                                 | string |
+| amount_in_usd            | The amount received in USD, during this snapshot.         | number |
+| amount_out_usd           | The amount sent in USD, during this given snapshot.       | number |
+| fees_usd                 | The fees collected in USD, during this snapshot.          | number |
+| chain_id                 | The standard chain ID where the maker resides.            | string |
+
+### Taker Snapshot
+
+List of takers in intent-based bridge protocols.
+
+| Property                | Description                                               | Type   |
+|-------------------------|-----------------------------------------------------------|--------|
+| timestamp                | The timestamp of the snapshot.                            | number |
+| user_address             | The address of the taker.                                 | string |
+| volume_usd               | The volume bridged in USD, in the given snapshot.         | number |
+| chain_id                 | The standard chain ID where the taker resides.            | number |
+
+### Token Snapshot
+
+List of takers in intent-based bridge protocols.
+
+| Property                | Description                                               | Type   |
+|-------------------------|-----------------------------------------------------------|--------|
```

Comment:
token_amount

Reply:
To clarify, this only covers pool based bridges? I also added `token_amount_usd`

Reply:
not that familiar with all type of bridges , but the goal here is to ensure we have the token amount column for all bridges

Reply:
Then I have added token amount and volume amounts and their corresponding USD fields.

- I belv you could have token amounts (or supplied amounts in pools), for pool based bridges
- All bridges have a volume bridged amount